docs/Configuration.md

```diff
@@ -145,11 +145,83 @@ oc create secret generic snow-secret \
 -n pelorus
 ```
 
+### Labels
+
+Labels are key/value pairs that are attached to objects, such as pods. Labels are intended to be used to specify identifying attributes of objects that are meaningful and relevant to users and Pelorus.
+
+The commit time, deploy time, and failure exporters all rely on labels to indentify the application that is associated with an object.  The object can
```

Comment:
NIT: 

is associated with an object.**<DOUBLE_SPACE_HERE>** The object

Reply:
hrm.. either you fixed it, or there were two spaces.. double checked :)